assets/style/style.css

```diff
@@ -407,41 +425,92 @@ body {
   gap: 15px;
 }
```

Comment:
- Please, let's improve the design on each card by improving the image height to show correctly as seen in the Figma design
- Please, let's use the correct color for the paragraph content in each card to make the design close to the Figma as much as we can. See image below:



*Figma design*
![Screenshot from 2023-03-08 14-17-44](https://user-images.githubusercontent.com/59397159/223723564-fb9127af-131b-433b-96ba-4df134c717ff.png)

*Your design*
![Screenshot from 2023-03-08 14-16-59](https://user-images.githubusercontent.com/59397159/223723575-4a67e219-807c-46b7-a5d2-7a578ff816b6.png)

Reply:
Thanks for your feedback dear @zieeco